spec/board_spec.rb

```diff
@@ -0,0 +1,54 @@
+require './lib/board'
+
+RSpec.describe Board do
+  let(:empty_arr) { ['', '', '', '', '', '', '', '', ''] }
+  let(:board) { Board.new }
+
+  context 'Create Board instances' do
+    it 'make sure that at the beginning we get an empty array' do
+      expect(Board.new.board).to eql(empty_arr)
+    end
+  end
+
+  context '#add_marker' do
```

Comment:
* [ ] Specs for each method should be wrapped in one `describe` block. As in example here https://github.com/rubocop-hq/rspec-style-guide#empty-lines-around-examples . ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/describe.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))

- change the context on line #13 to be a  `describe` block.

Reply:
thank you for this notice I never know that :)